src/vt/configs/arguments/args.cc

```diff
@@ -185,6 +193,241 @@ std::tuple<int, std::string> parseArguments(
   return std::make_tuple(-1, std::string{});
 }
 
+void addYamlConfig(CLI::App& app, AppConfig& appConfig) {
+  auto yaml_description = "Read in a yaml config file for VT";
+  app.add_option("--vt_input_config_yaml", appConfig.vt_input_config_yaml, yaml_description);
+}
+
+void parseYaml(AppConfig& appConfig, std::string inputFile) {
+  // Read in the YAML configuration file
+  auto yaml_input = YAML::LoadFile(inputFile);
+
+  // Output control
+  YAML::Node output_control = yaml_input["Output Control"];
+  appConfig.vt_color = output_control["Color"].as<bool>(true);
```

Comment:
Consider what happens when user passes `--vt_color=false` in the command line.

Use current value of configuration variable as fallback:
```suggestion
  appConfig.vt_color = output_control["Color"].as<bool>(appConfig.vt_color);
```
This should make sense whatever order of precedence we end up with.

Reply:
Fixed